packages/breadboard-web/index.html

```diff
@@ -9,7 +9,6 @@
 </head>
 <body>
   <script type="module">
-    import * as pkg from "./package.json";
```

Comment:
Isn't this used to get the version to print in the console?

Reply:
oops that got in from a branch switch